resources/build/builder.inc.sh

```diff
@@ -0,0 +1,680 @@
+#!/usr/bin/env bash
+#
+# This script contails utilities for builder_script calls
+#
+
+function _builder_findRepoRoot() {
+    # See https://stackoverflow.com/questions/59895/how-to-get-the-source-directory-of-a-bash-script-from-within-the-script-itself
+    # None of the answers are 100% correct for cross-platform
+    # On macOS, requires coreutils (`brew install coreutils`)
+    local SCRIPT=$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")
+    REPO_ROOT=$(dirname $(dirname $(dirname "$SCRIPT")))
+    readonly REPO_ROOT
+}
+
+_builder_findRepoRoot
```

Comment:
Move this line, and the call to `_builder_setBuildScriptIdentifiers` to after the function def for `_builder_setBuildScriptIdentifiers`

Reply:
Now in a new function _builder_init()